content/applications/websites/ecommerce/managing_products/products.rst

```diff
@@ -0,0 +1,197 @@
+- :guilabel:`Cart`: :guilabel:`Buy Now` adds a checkout button taking the customer directly to the
+  checkout page; :guilabel:`Wishlist` allows to add the product to a wishlist;
+- :guilabel:`Specification`: allows you to select where the **specification** category is displayed.
+   This only applies to products with **variants**.
+
+.. note::
+   - To allow **wishlists**, the option must be enabled in :menuselection:`Configuration -->
+     Settings --> Shop - Products --> Wishlists`;
+   - To access the :guilabel:`Variants` options, the :guilabel:`Product Variants` option must first
+     be enabled under :menuselection:`Configuration --> Settings --> Shop - Products`. You can learn
+     more about variants :doc:`here <../../../sales/sales/products_prices/products/variants>`.
+
+Layout
+------
+
+Within the same :guilabel:`Customize` tab as the :ref:`features <ecommerce-features>`, the layout
+configuration can be changed according to your needs.
+
+- :guilabel:`Images Width`: changes the *width* of the image product displayed on the page;
+- :guilabel:`Layout`: the :guilabel:`Carousel` layout displays a large, main image with smaller ones
+  underneath; whereas the :guilabel:`Grid` displays four images in a square layout (see pictures
+  below);
+- :guilabel:`Image Zoom`: if the zoom occurs on :guilabel:`Pop-up on Click`, when hovering over the
+  image (:guilabel:`Magnifier on hover`), on :guilabel:`Both`, or :guilabel:`None`;
+- :guilabel:`Image Spacing`: how far apart the images are;
+- :guilabel:`Columns`: how many columns of images are displayed;
+- :guilabel:`Main Image`: click :guilabel:`Replace` to change the product's main image;
+- :guilabel:`Extra Images`: click :guilabel:`Add` or :guilabel:`Remove all` to add or remove product
+  images. You can also add images and videos via URL.
+
+.. note::
+   Images must be either PNG or JPG and at least 1042px in height or width. To trigger the zoom, the
+   image has to be bigger than 1024x1024
+
+.. image:: products/products-layout.png
+   :align: center
+   :alt: Product images layout
+
+Add content
+-----------
+
+You can use **building blocks** to add content to your product page. These blocks can be used to add
+extra text and picture galleries, features such as call to actions or comparisons, etc. Depending on
+*where* you drop the **building block**, it may be available either on the *product page* only, or
+on the *whole* website.
+
+.. image:: products/products-blocks.png
+   :align: center
+   :alt: Building blocks on product page
+
+Product configuration
+=====================
+
+Multiple languages
+------------------
+
+If multiple languages are available on your website and you wish to have the product's information
+translated, it is necessary to encode these translated information in the **product's form**. Fields
+with multiple languages available are identifiable by their abbreviation language (ex. :abbr:`EN
+(English)`) next to their field.
```

Comment:
It is because (as written) found in the **product's form** and not the **product's page**. I changed the wording so it is less confusing. Ideally, there'd be a :ref: to explain how to check that, but the page actually doesn't exist. I'll add or a note or something in the meantime